src/ansi.rs

```diff
@@ -715,6 +718,73 @@ bitflags! {
     }
 }
 
+/// Construct escape sequences for regular keys, like `1`, when
+/// modified by `Shift-`, `Control-`, `Alt-` or `Meta-`modifiers.
+/// This doesn't apply to the named keys on the keyboard, like `F1`.
+/// The escape sequence also doesn't apply to defined unicode input, like
+/// cyrillic chars.
```

Comment:
The "defined" here is confusing. What does "defined unicode input" mean?

Is the following correct?

```
/// This only applies to keys corresponding to ascii characters.
```

If it is, this would be simpler and it would also remove the need for the `F1` comment.